homeassistant/components/proximity/sensor.py

```diff
@@ -69,10 +70,31 @@ async def async_setup_entry(
         for description in SENSORS_PER_PROXIMITY
     ]
 
+    tracked_entity_descriptors = []
+
+    entity_reg = er.async_get(hass)
+    for tracked_entity_id in coordinator.tracked_entities:
+        if (entity := entity_reg.async_get(tracked_entity_id)) is not None:
+            tracked_entity_descriptors.append(
+                {
```

Comment:
The item could be eg a named tuple for easier access. Not required change though.

Reply:
https://docs.python.org/3/library/typing.html#typing.NamedTuple

Ok to save for a follow up.

Reply:
ok, will prepare a follow up today evening/night 👍 
the other two comments has been applied.

Reply:
#109249